wled00/const.h

```diff
@@ -280,6 +280,9 @@
 #define TYPE_LPD8806             52
 #define TYPE_P9813               53
 #define TYPE_LPD6803             54
+
+#define TYPE_HUB75MATRIX         100 // 100 - 110
```

Comment:
This may need revisiting as it somehow clashes with @Aircoookie 's bit oriented approach.

Reply:
Can you explain this comment further?

Reply:
I tried to hide the fact that this was a bitmasked value in 0.15 macros (check marcos in 0.14 or previous). But I would still like it to follow the original idea of particular bit defining LED type if possible.
There is no need for you to do anything at this point. It was more of a reminder for myself.

Reply:
Once we have the proper UI changes to support non-standard config options then we should only need one maybe two IDs to handle all the different panel setups. At this point, we can swap to the proper ID(s) rather than my placeholder of 101 -> 110

Reply:
Network types are 80-95 and 2 pin digital 48-63, so IMO Hub75 types can span 64-71 (8 types) or 64-79 (16 types). Would that be ok?
I would not consider Hub75 to be a *virtual* type as it needs GPIO pins.
And if you want additional *virtual* types those can go all the way to 126. I would reserve 127 for usermod type as discussed elsewhere. The usermod type *can* reserve GPIO pins as usermod instead of as a bus.